core/dbt/contracts/rpc.py

```diff
@@ -472,6 +471,9 @@ def from_result(
     ) -> 'PollRunOperationCompleteResult':
         return cls(
             success=base.success,
+            results=base.results,
+            generated_at=base.generated_at,
+            elapsed_time=base.elapsed_time,
```

Comment:
I added these fields to the response because mypy was sad about some types further up the chain. We could also remove some of these fields from the other RPC Results instead